### DISCUSSION TITLE
Submit form on blur event

### DISCUSSION THREAD

**Discussion by discordDOTtest** [jiripudil/Naja#350] [Support] [answered]
Hi, I'm learning how to use Naja. I use this code to submit form on blur by Naja

```
$(document).ready(function(){
    $(document).on('blur', '.input-listening-for-blur-event', function(){
        $(this).closest('form').find('.submit-button-with-ajax-class').click();
    });
});
```

It works. I have this question: is it legit or not to submit "naja ajax" form like this?

**Comment by jiripudil** [comment, marked as answer]
Hi, sure. Naja has [helper methods](https://naja.js.org/#/ui-binding?id=manual-dispatch) to submit a form or programmatically "click" an element, but what you're doing is absolutely fine too as long as it dispatches the click event for Naja to pick up.

**Reply by discordDOTtest** [reply]
Thank you!
